root/programs/C#/Frameworks/Infrastructure/Public/Util/CodeSigningXML.cs

```diff
@@ -0,0 +1,274 @@
+﻿//**********************************************************************************
+//* Copyright (C) 2007,2016 Hitachi Solutions,Ltd.
+//**********************************************************************************
+
+#region Apache License
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. 
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#endregion
+
+//**********************************************************************************
+//* クラス名        ：CodeSigning
```

Comment:
@yusukemaegawa 

I changed the class name as follows.
- CodeSigning ---> DigitalSign
- CodeSigningXML ---> DigitalSignXML
- CodeSigningXML ---> DigitalSignX509